docs/mixpanel.md

```diff
@@ -0,0 +1,33 @@
+### What is Mixpanel?
+It is an analytic tool that gets you insighst on how users interact with your digital product. These insights will help enable you to make decisions on how your users can get the best out of your digital productand and eventually retain them.
+
+Get an overview of the Mixpanel: https://developer.mixpanel.com/docs/what-is-mixpanel 
+
+### How to set it up?
+
+- On the Mixpanel dashboard, create a project and get the project token.
+
+- On your machine, navigate to your project directory and in the terminal, run the following command `npm install --save mixpanel-browser` or `yarn add mixpanel-browser` to install the package
+- After installing the package, open your text editor,
+    - In App.js, start using Mixpanel in your project:
+    ``` 
+    import mixpanel from 'mixpanel-browser';
+    // or with require() syntax:
+    // const mixpanel = require('mixpanel-browser');
+
+    // Enabling the debug mode flag is useful during implementation,
+    // but it's recommended you remove it for production
+    mixpanel.init('YOUR_TOKEN', {debug: true}); 
+    mixpanel.track('Sign up'); 
+    ```
+
+### How to check for your events
+- Go back to Mixpanel project dashboard Live View(now Events) to see all data associated to an event.
```

Comment:
Please add more instructions here. For instance, maybe a screenshot of what that screen might look like? And also please call out that they will need to test an event, ie, if they set an event for clicking the button, they'll need to actually click a button in order to see that event register.